src/pages/v2/p/[projectId]/components/V2Dashboard.tsx

```diff
@@ -223,7 +223,7 @@ export default function V2Dashboard({
   return (
     <V2ProjectContext.Provider value={project}>
       <div style={layouts.maxWidth}>
-        <V2Project />
+        <V2Project selectedPage="info" />
```

Comment:
**suggestion** can we use the URL as a single source of truth? In other words, can we use the URL alone to determine the render state?